### ISSUE TITLE
The map requires fairly new browser versions

### ISSUE THREAD

**Issue by nautofon** [truckermudgeon/maps#27]
In an unrelated discussion on the forum, [it's been reported](https://forum.scssoft.com/viewtopic.php?p=1995298#p1995298) that the map doesn't work in a mobile Opera version "a bit behind" the most recent one. The reported error message was:
```
Unexpected Application Error!
Object.groupBy is not a function
```

`groupBy` is fairly new ([caniuse](https://caniuse.com/groupBy) / [mdn](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/groupBy#browser_compatibility)). Most browsers only added it a year ago. Unfortunately, many users don't update that often.

While I think it's [reasonable to not support bad browsers](https://alistapart.com/article/tohell/), the app could perhaps fail a bit more gracefully than just throwing an "unexpected error".

**Comment by nautofon** [comment]
Oh I see now you already worked to address this while I was typing up the report. 😆

**Comment by truckermudgeon** [comment]
Thanks for the report! I did see the original issue when browsing the forums as I waited for the Greece DLC 😅 

Fixed in 326fe51226e6858a2e7ab6e7a20665bbaaed9479
